<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeaMenu</class>
 <widget class="QDialog" name="TeaMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="edt_S1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_Total">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_S5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_S4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>总分</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Over">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>本卷判毕</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>编程得分</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edt_Paper">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>461</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_S6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_S3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_S2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Exe">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cal">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>计算总分</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_New">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>判新卷</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_File">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Doubt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>学生成绩问题反馈情况</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>390</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>成绩保存文件目录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_Paper">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>改错</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>编程</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_Paper">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示卷面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Score">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>成绩管理界面</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spb_StuID">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>3210.000000000000000</double>
   </property>
   <property name="maximum">
    <double>3220.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>改错得分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pic/seu.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
